docs/pages/application-access/getting-started.mdx

```diff
@@ -43,9 +43,9 @@ $ docker run -d -p 3000:3000 grafana/grafana
 ## Step 2/3. Install and configure Teleport
 (!docs/pages/includes/permission-warning.mdx!)
 
-On your Application Service host, download the latest version of Teleport for
-your platform from our
-[downloads page](https://goteleport.com/teleport/download).
+On your Application Service host install the appropiate package for your platform.
+
+(!docs/pages/includes/install-linux.mdx!)
```

Comment:
In this and other pages in this PR, should we copy the content of the `install-linux.mdx` partial into a separate partial that also includes `TabItem`s for Docker and Helm instructions? 

These could either be additional `TabItems` in the same `Tabs` component (we're adding support for more tabs in the `Tabs` component, so this should only look odd temporarily) or a separate `Tabs` component for a containerized installation.

We could also explicitly call for a Linux server like we do in the Desktop Access guide. (Or we could update the Desktop Access guide to remove the requirement for a Linux server.)

Reply:
Ah, that's a good point. It might be easiest to add a requirement for a Linux server in the Prerequisites, then use the `install-linux.mdx` partial.